### PR TITLE
Add service add/remove-secret sub-commands

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

File: cli/lib/kontena/cli/service_command.rb [hunk 1]
```diff
@@ -31,10 +35,14 @@ class Kontena::Cli::ServiceCommand < Clamp::Command
...
   subcommand "logs", "Show service logs", Kontena::Cli::Services::LogsCommand
   subcommand "stats", "Show service statistics", Kontena::Cli::Services::StatsCommand
+  subcommand "monitor", "Monitor", Kontena::Cli::Services::MonitorCommand
+
   subcommand "envs", "Show environment variables", Kontena::Cli::Services::EnvsCommand
   subcommand "add-env", "Add environment variable", Kontena::Cli::Services::AddEnvCommand
   subcommand "remove-env", "Remove environment variable", Kontena::Cli::Services::RemoveEnvCommand
-  subcommand "monitor", "Monitor", Kontena::Cli::Services::MonitorCommand
+
+  subcommand "add-secret", "Add secret from Vault", Kontena::Cli::Services::AddSecretCommand
```
Comment (hans-d) [inline cli/lib/kontena/cli/service_command.rb]: why not as `'kontena service secret add`  and `kontena service secret remove`? (see #521 and #512)

Comment (jakolehm) [inline cli/lib/kontena/cli/service_command.rb]: simple reason: we will refactor sub-commands to new (better) style in a one PR .. target for that is post 0.12

**Comment by jakolehm** [comment]
ping @nevalla @miskun

**Comment by miskun** [comment]
I'm having same thinking as @hans-d but I guess we can refactor sub-commands names also later. Since the sub commands naming plan is still "under construction" I'm ok to go ahead now with this implementation. +1

**Comment by nevalla** [comment]
Yes agree with @miskun. Let's go now with this implementation.

**Pull request merged**